app/models/volunteer.rb

```diff
@@ -82,11 +82,6 @@ def self.state_collection
                           :dancing, :health, :cooking, :excursions, :women, :teenagers,
                           :children, :other_offer].freeze
   REJECTIONS = [:us, :her, :other].freeze
-  AVAILABILITY = [:flexible, :morning, :afternoon, :evening, :workday, :weekend].freeze
-
-  def self.human_boolean(boolean)
-    boolean ? I18n.t('simple_form.yes') : I18n.t('simple_form.no')
```

Comment:
why did you remove this? 
I thought this was used for example in the checklist which puts `false = no`, `true = yes`